src/AppBundle/Entity/RepositoryStarRepository.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\EntityRepository;
+
+class RepositoryStarRepository extends EntityRepository
+{
+    public function create(Repository $repository, User $user)
```

Comment:
Not really a repository method.
IMHO, should be a static method inside the RepositoryStar entity